src/TcBlack/ObjectDefinition.cs

```diff
@@ -138,7 +153,7 @@ private TcObject TokenizeFunctionBlock()
         private TcObject TokenizeMethodOrProperty()
         {
             string entityType = @"\s*(FUNCTION|METHOD|PROPERTY)\s*";
-            string accessModifier = @"(PRIVATE|PUBLIC|PROTECTED|INTERNAL)?\s*";
+            string accessModifier = @"(PRIVATE|PUBLIC|PROTECTED|INTERNAL|FINAL|ABSTRACT)?\s*";
```

Comment:
```suggestion
            string accessModifier = 
                @"(PRIVATE|PUBLIC|PROTECTED|INTERNAL|FINAL|ABSTRACT)?\s*";
```

I'm trying to also maintain the 88 line length limit in the code itself. See for more info https://github.com/Roald87/TcBlack/blob/master/CONTRIBUTING.md#use-a-consistent-coding-style

Reply:
Sure, is there a C# Black that can be added to the project?

Reply:
I haven't found a free one unfortunately. There is https://www.jetbrains.com/resharper/, but you have to pay for that one.